gpkit/interactive/sankey.py

```diff
@@ -13,6 +14,9 @@
 
 INSENSITIVE = 1e-2
 
+if sys.version_info >= (3, 0):
+    unichr = chr  # pylint: disable=redefined-builtin,invalid-name
```

Comment:
wondering if it would be better to define `version_insensitive_unichr` (holy moly hopefully someone comes up with a better name), but anyway use that everywhere in this file instead of redefining the builtin?